holochain.go

```diff
@@ -612,6 +614,33 @@ func (h *Holochain) Send(proto Protocol, to peer.ID, t MsgType, body interface{}
 	return
 }
 
+func (h *Holochain) Sign(doc []byte) (sig []byte, err error) {
```

Comment:
needs a comment before it like:
//Sign uses the agent' private key to sign the contents of doc